target_snowflake/db_sync.py

```diff
@@ -408,9 +411,17 @@ def query(self, query, params=None, max_records=0):
                 else:
                     queries = [query]
 
+                qid = None
+
                 for q in queries:
-                    self.logger.debug("Running query: {}".format(q))
+                    # support for a QID name parameter in case we want to pass the last query id to the current query
+                    q = q.format(QID=qid)
+
+                    if safe_to_log:
+                        self.logger.info("Running query: %s", q)
```

Comment:
if we add `safe_to_log` then we won't be able to log queries dynamically by setting log level in conf. What's the benefit of this condition?

Reply:
So there are two reasons:
* if we want to use the `DBSync.query` method for queries that have some sort of secrets and we don't want them to be logged regardless of the log level.
* Currently the log level is debug, and these show followed by select are queries that I guess would be nice to have readily available in the logs without having to change the log level from info to debug, as enabling debug would risk increasing the log file size thus file system issues.

wdyt?

Reply:
ok, but the default value of `safe_to_log` is `False` and we'll see logs only if the query called by  the `get_tables` or `get_table_columns` functions even if we set the log level to debug.

Can we set the default `safe_to_log` to `True`?

Reply:
Sure

Reply:
Done.